src/main/java/racingcar/controller/RacingCarController.java

```diff
@@ -0,0 +1,26 @@
+package racingcar.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import racingcar.domain.Car;
+import racingcar.dto.RacingInput;
+import racingcar.dto.RacingResult;
+import racingcar.domain.RacingCar;
+import racingcar.dao.RacingCarDao;
+
+import java.util.List;
+
+@RestController
+public class RacingCarController {
+
+    @PostMapping(value = "/plays", consumes = "application/json")
+    public ResponseEntity startGame(@RequestBody RacingInput racingInput){
```

Comment:
```suggestion
    public ResponseEntity<RacingResult> startGame(@RequestBody RacingInput racingInput){
```

`ResponseEntity` 객체는 HTTP 응답을 나타내는 객체입니다. 그러나, 반환하는 데이터가 좀 더 구체적으로 명시되는 것이 좋을 수 있습니다.


```
{
    "winners": "워니",
    "racingCars": [
        {
            "name": "워니",
            "position": 9
        },
        {
            "name": "제이슨",
            "position": 7
        },
        {
            "name": "브라운",
            "position": 3
        },
    ]
}
```

의 형태를 가지는 객체를 반환해 준다면, 하위 프로세스를 탐색해보지 않더라도, 
`RacingResult`만 확인 해본다면 어떠한 포맷을 반환해주는구나! 를 쉽게 알수 있지않을까요!? 🤔

Reply:
적용완료!